pkg/azuredisk/azuredisk_v2.go

```diff
@@ -33,12 +35,15 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Consider given this the alias `apiRuntime` to avoid ambiguity with the [Go `runtime` package](https://pkg.go.dev/runtime).

Reply:
Done :)